libsodium-sys/build.rs

```diff
@@ -346,29 +347,22 @@ fn build_libsodium() {
     fs::create_dir_all(&source_dir).unwrap();
 
     // Copy sources into build directory
-    let cp_status = if target.contains("msvc") {
-        Command::new("xcopy")
-            .arg("libsodium")
-            .arg(&source_dir)
-            .args(&["/s", "/e", "/i", "/q", "/y"])
-            .status()
-    } else {
-        Command::new("cp")
-            .arg("-r")
-            .arg("libsodium/.")
-            .arg(&source_dir)
-            .status()
-    };
-
-    match cp_status {
-        Ok(status) if status.success() => (),
-        Ok(status) => {
-            panic!("Failed to copy sources into build directory: {}", status);
+    // Skip .git because it's marked read-only and that causes problems re-building
+    for entry in walkdir::WalkDir::new("libsodium")
+        .into_iter()
+        .filter_entry(|e| e.file_name() == OsStr::new(".git"))
```

Comment:
```suggestion
        .filter_entry(|e| e.file_name() != OsStr::new(".git"))
```
This needs to be flipped. To reproduce the error try to delete `target/` and rebuild.

Reply:
Whoops, that was a bit silly. Pushed an update, thanks.